CustomDictionary.xml

```diff
@@ -71,6 +71,7 @@
       <Word>SingletonAttribute</Word>
       <Word>Prefetch</Word>
       <Word>ScopeId</Word>
+      <Word>Poco</Word>
```

Comment:
I'm getting warnings now too. But I have the change in CustomDictionary... so looks like styleCop is broken.